users/forms.py

```diff
@@ -11,15 +14,17 @@ class Meta:
             'password': forms.PasswordInput()
         }
 
-    def save(self):
-        data = self.cleaned_data
-
-        user = User(
-            email=data['email'],
-            username=data['username'],
-        )
-        user.set_password(data['password'])
+    def save(self, request):
+        user = super().save(commit=False)
+        user.set_password(self.cleaned_data['password'])
         user.is_active = False
         user.save()
 
-        return user
+        SendActivationEmail(
+            user=user,
+            site=get_current_site(request),
+            token_generator=default_token_generator,
+            use_https=request.is_secure(),
+        ).call()
+
+        self.user = user
```

Comment:
Zrobione - zostawiam to pole user tylko po to zeby nie musiec zmieniac `context` w odpowiedzi.